Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OuterTune/App - OuterTune](https://hosted.weblate.org/projects/outertune/app/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/outertune/app/horizontal-auto.svg)

lucasmz-dev commented:
I think I'm gonna disable automatic upstream updating, force pushes are common in this repo and it's just easier that way

DD3Boh committed:
app: settings: Hide arrange tabs setting with the new interface layout

 * This is only supposed to work with the old tabs and not with the
   new layout.

Co-authored-by: Davide Garberi <dade.garberi@gmail.com>

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (140 of 140 strings)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (138 of 138 strings)

Co-authored-by: Максим Горпиніч <maksimgorpinic2005a@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/uk/
Translation: OuterTune/App - OuterTune

lucasmz-dev committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (140 of 140 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 96.3% (133 of 138 strings)

Translated using Weblate (Japanese)

Currently translated at 100.0% (138 of 138 strings)

Translated using Weblate (French)

Currently translated at 100.0% (138 of 138 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (138 of 138 strings)

Co-authored-by: LucasMZ <git@lucasmz.dev>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/fr/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ja/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/outertune/app/zh_Hans/
Translation: OuterTune/App - OuterTune

reocat committed:
Translated using Weblate (Russian)

Currently translated at 100.0% (140 of 140 strings)

Co-authored-by: reocat <ng.ct_ml@tuta.io>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/ru/
Translation: OuterTune/App - OuterTune

Mickael81 committed:
Translated using Weblate (French)

Currently translated at 100.0% (140 of 140 strings)

Co-authored-by: Mickaël Binos <mickaelbinos@outlook.com>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/fr/
Translation: OuterTune/App - OuterTune

Commit:
Translated using Weblate (Azerbaijani)

Currently translated at 37.1% (52 of 140 strings)

Co-authored-by: Femini <nizamismidov4@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/outertune/app/az/
Translation: OuterTune/App - OuterTune
